staging/src/k8s.io/sample-apiserver/docs/minikube-walkthrough.md

```diff
@@ -1,56 +1,67 @@
 # Minikube walkthrough
 
 This document will take you through setting up and trying the sample apiserver on a local minikube from a fresh clone of this repo.
+You can optionally run and develop the sample apiserver on your local machine and use minikube to host the etcd.
 
-## Pre requisites
+## Prerequisites
 
-- Go 1.7.x or later installed and setup. More infomration can be found at [go installation](https://golang.org/doc/install)
+- Go 1.7.x or later installed and setup. More information can be found at [go installation](https://golang.org/doc/install)
 - Dockerhub account to push the image to [Dockerhub](https://hub.docker.com/)
-
-## Install Minikube
-
-Minikube is a single node Kubernetes cluster that runs on your local machine. The Minikube docs have installation instructions for your OS.
-- [minikube installation](https://github.com/kubernetes/minikube#installation)
-
+- Minikube to run a single node Kubernetes cluster on your local machine. The Minikube docs have [installation instructions](https://github.com/kubernetes/minikube#installation) for your OS.
+- kubectl for running commands against the minikube Kubernetes cluster. [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 
 ## Clone the repository
 
-In order to build the sample apiserver image we will need to build the apiserver binary.
+In order to build or develop the sample apiserver image we'll need to clone the repository.
 
-```
+```bash
 cd $GOPATH/src
 mkdir -p k8s.io
 cd k8s.io
```

Comment:
These 3 commands could be simplified to
```bash
mkdir $GOPATH/src/k8s.io
cd $GOPATH/src/k8s.io
```

Reply:
@php-coder nice catch. I'll fix it in subsequent PR